### ISSUE TITLE
extend time between each update check

### ISSUE THREAD

**Issue by Khidreal** [digimezzo/dopamine-windows#148]
I think an interval of 15 minutes between each update check is too much... 
not always people have an unlimited connection, sometimes we may be using a limited connection or a really slow connection.. I can speak for myself, 2 years ago I was having an internet connection that after 2GB, would reduce it's speed from 30Mb/s to 128Kb/s. sometimes, even my antivirus sending an update check would break any download I was making... and this request is thinking about it. 
also, having Dopamine sending data packets each 15 minutes well, it's just not necessary in my opinion.

**Comment by Khidreal** [comment]
I even thought a bit more about this, and I realise that there is not even need to send an update check from time to time. I mean, nobody will stay with Dopamine open forever, so just the ability to check for updates on startup it's enough, just my opinion.

**Comment by digimezzo** [comment]
Agreed. I'll make the "from time to time" configurable (length, and enabled/disabled).
